wasm-rpc/src/lib.rs

```diff
@@ -304,6 +402,576 @@ fn build_tree(node: &WitNode, nodes: &[WitNode]) -> Value {
     }
 }
 
+// An efficient read representation of a value with type information at every node
+// Note that we do an extra annotation of AnalysedType at complex structures to fetch their type information with 0(1)
+// However the typical use of TypeAnnotatedValue is similar to using a Json/Yaml etc
```

Comment:
This comment does not make much sense to me and is confusing

Reply:
Ah! Forgot

Reply:
Removed